lib/shopify_api/resources/assigned_fulfillment_order.rb

```diff
@@ -0,0 +1,8 @@
+module ShopifyAPI
+  class AssignedFulfillmentOrder < Base
+    def self.all(options = {})
+      assigned_fulfillment_orders = super(options)
+      assigned_fulfillment_orders.map { |afo| FulfillmentOrder.new(afo.as_json) }
```

Comment:
Could we use `attributes` instead of `as_json`?

Reply:
In this instance `attributes` works just fine.